lab1/lab1.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Homepage</title>
+</head>
+
+<link href = "lab1.css" rel = "stylesheet" type= "text/css"> 
+
+	<style>
+.boxes {
+  display: inline-block;
+  width: 360px;
+  height: 360px;
+}
+#leftBox {
+  float: left;
+}
+#rightBox {
+  float: right;
+}
+</style>
+
+<body>
+	<h1>Incremental Game</h1>
+
+	<div style="height:150px;width:1000px;background-color : grey; border:5px solid #ccc;font:20px/26px  Georgia, Garamond, Serif;overflow:auto;">
+		<p> Storyline 1 </p>
+		<p>storyline 2 </p>
+		<p>storylie 3  </p>
+		<p>storyline 4 </p>
+		</div>
+<h1>{Resource name} : </h1> 
+<div class="wrapper">
+    <button class="button">action</button>
+</div>
+<div style="height:300px;width:400px;border:5px solid #ccc;font:20px/26px Georgia, Garamond, Serif;overflow:auto;"> 
```

Comment:
inline style should be avoided. you should create a class with all the properties you defined here under style.css and add class to this element.